### DISCUSSION TITLE
We need more information about the best way to use calibration

### DISCUSSION THREAD

**Discussion by julie777** [OrcaSlicer/OrcaSlicer#687] [Q&A]
The Wiki discusses how to run and interpret the tests. It doesn't discuss what order to run tests, which tests are most important, etc. For those of us who are less advanced we could use more information.

I think the idea of auto-cal is a killer feature!

**Comment by DuckTown407** [comment]
You should run PA, then EM, and that is mostly all you need.

**Reply by julie777** [reply]
Why not run all and make it as good as it can be?

**Reply by DuckTown407** [reply]
Temp is usually just unecessary - within listed temp ranges for filament types is usually fine. I really don't run max flow rate testing either because the max flowrate for the X1 is known to be around 19mm3/s and I don't really want to run close to that any way.

**Comment by Kaan88** [comment]
I find that EM reduces PA, so it seems the order should be PA, EM then PA again.

**Reply by otede** [reply]
What the hell is EM calibration?!

**Comment by julie777** [comment]
I ran the tests in the order of the wiki page documentation.
The flow rate test seemed to work well although the individual pieces all had the corners come loose and curl up.

I tried the PA line test and had some problems with a couple pulling loose and following the nozzle around. I decide to skip this version. BTW, this is the first time I have ever needed to use a scraper to release the model from the bed. The lines weren't too bad, but the tiny numbers were like popcorn and flew all over the room.

Next I rand the PA test with the tower. With the neon pink filament I happen to be using it is much harder to see the details than in the wiki photos. Regardless, the wiki doesn't describe what to look for and how to determine what is best. It shows a photo of the corner with the sharpest angle and it seems like there is a transition from over extrusion to under extrusion that is being chosen. I see the change on my print, but the transition point is not clear.

Looking at the back of the model, the seam really shows wide variation, but again, what am I looking for to make the choice.

Then I printed the temp tower. I looked at 4 things and picked different temperature for each. 
- the end overhang - best at 210
- stringing of the bridge underside - best at 230
- the post in the window - perfect at 225
- the top of the circular hole forming the window. - cannot see it well enough to really tell

My thoughts about the temp tower
- using a tower looks cool, but it means that every temperature prints in a different environment. While printing at 195 mine turned into a spaghetti mess. Probably due to temp, but likely made worse by movement cause by height. **I suggest printing as separate objects** I have no idea how this affected the accuracy of the various temps.
- The bridging would be easier to examine if the vertical gaps was larger.

Finally, I don't know how to examine the object and decide which defects are most important and how to choose the best temp.

I do love auto-cal. :smiley:

**Comment by nerdo** [comment]
Before I got into the documentation, I started doing a temp tower test first. I do agree that I have some concerns about the height affecting the results - seems like it will naturally be cooler the further away it gets from the bed. I'm not sure why these aren't done separately, but I don't have any facts to back this up. Just a hunch.

Anyway, i ended up here because I have the same question about what order the tests should run in. Since I already ran the temp tests, I'm going in the order the doc says after setting the temperature, so flow rate, then PA, and will do a max flow rate afterwards. Not sure if that order is fine or not, but we shall see 🤷🏾 . It would be nice to get some clarity on this.

**Reply by LJKraus** [reply]
I'm still uncertain about the correct, or optimal order to run the calibrations.  Surely temperature, PA and flow rate all interact, so the first test might be negatively affected by a setting changed after the second or third test.  I really don't want to run through multiple iterations of all the tests to feel confident I'm getting the best results.

Maybe I would not have so many doubts if the wiki and the slicer's list of tests were in the same order?

**Comment by tsmith35** [comment]
I've found that the clone CHT nozzle design is quite good at increasing my flow rates. I initially became interested in the design to improve the performance with PETG, to avoid some of the temperature fluctuations in order to better accommodate the smaller "sweet spot" of PETG. So the flow rate testing is still useful to me. That was all I had to say. :-)
